web/deploy/terraform/modules/iam/policies/assume-role.json

```diff
@@ -4,13 +4,14 @@
         {
             "Effect": "Allow",
             "Principal": {
-                "Federated": "arn:aws:iam::123456123456:oidc-provider/token.actions.githubusercontent.com"
+                "Federated": "arn:aws:iam::390844787521:oidc-provider/token.actions.githubusercontent.com"
```

Comment:
If this is the sandbox account, make sure to change it for the real one

Reply:
Actually, let's read this from a secret or environment variable

Reply:
[Fixed](https://github.com/smokestacklightnin/osm/pull/9/commits/fef0b39eafcef52f00bf82533649e8f05f56efe3)